features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/postgresql.sql

```diff
@@ -598,6 +598,21 @@ CREATE TABLE SP_TEMPLATE (
   CONSTRAINT SP_TEMPLATE_CONSTRAINT UNIQUE (TENANT_ID, NAME)
 );
 
+DROP TABLE IF EXISTS SP_TRUSTED_APPS;
+DROP SEQUENCE IF EXISTS SP_TRUSTED_APPS_SEQ;
+CREATE SEQUENCE SP_TRUSTED_APPS_SEQ;
+CREATE TABLE SP_TRUSTED_APPS (
+			ID INTEGER DEFAULT NEXTVAL('SP_TRUSTED_APPS_SEQ'),
+			SP_ID INTEGER,
+			PLATFORM_TYPE VARCHAR(255) NOT NULL,
+            APP_IDENTIFIER VARCHAR(255) NOT NULL,
+            THUMBPRINTS VARCHAR(2048),
+            IS_FIDO_TRUSTED BOOLEAN DEFAULT FALSE,
+			TENANT_ID INTEGER DEFAULT -1,
```

Comment:
Is there a specific reason to have a default value for the TENANT_ID? if not lets mark it as NOT NULL and remove the default. Check this in other db scripts as well.